contracts/abstract/PrizeSplit.sol

```diff
@@ -139,6 +139,19 @@ abstract contract PrizeSplit is OwnableUpgradeable {
       return (amount * percentage).div(1000);
   }
 
+  /**
+  * @notice Require the prize split percentage amount does not exceed 100%.
+  * @dev Require the prize split percentage amount does not exceed 100% with the current prize splits.
+  */
+  function _totalPrizeSplitPercentageAmount() internal view returns (uint256) {
+    uint256 _tempTotalPercentage;
```

Comment:
Could just be called `_totalPercentage`, it's pretty explicite that this is a temporary variable only used in this internal function.

Reply:
For the time being I will probably just keep as is, since it's more descriptive of what's occurring.